### PR TITLE
Add support for rendering entities with a simple RGBA value.

### PR DESCRIPTION
**Pull request by Xaeroxe**
I wanted to programatically control a sprite's alpha value and I realized there wasn't an easy way to do that.  I figured while I was at it, why not also allow all four RGBA channels to be controlled like this?

### CODE REVIEW

File: amethyst_renderer/src/pass/shaders/vertex/sprite.glsl [hunk 1]
```diff
@@ -39,8 +45,8 @@ void main() {
     float tex_v = positions[gl_VertexID][1];
 
     vec2 uv = pos + tex_u * dir_x + tex_v * dir_y;
-    tex_uv = texture_coords(vec2(tex_u, tex_v), u_offset, v_offset);
-
+    vertex.tex_uv = texture_coords(vec2(tex_u, tex_v), u_offset, v_offset);
+    vertex.color = color;
     vec4 vertex = vec4(uv, depth, 1.0);
```
Comment (azriel91) [inline amethyst_renderer/src/pass/shaders/vertex/sprite.glsl]: does glsl support shadowing?
I thought it would clash

Comment (Xaeroxe) [reply amethyst_renderer/src/pass/shaders/vertex/sprite.glsl]: This isn't technically shadowing, as `vertex.color` is in a different namespace from `color`.

Comment (Xaeroxe) [reply amethyst_renderer/src/pass/shaders/vertex/sprite.glsl]: But to your other comment, I tested the output of all these shaders.

**Comment by Xaeroxe** [comment]
@azriel91 changelog entry added, thanks for the reminder!

File: amethyst_renderer/src/pass/shaders/vertex/sprite.glsl (hunk 1 above)
Comment (azriel91) [reply amethyst_renderer/src/pass/shaders/vertex/sprite.glsl]: ah cool

**Comment by Xaeroxe** [comment]
I'm doubtful we're going to get a request for changes in the next 3 hours, so I'm just going to merge this.  Happy holidays everyone!

bors r+

**Pull request merged**